src/core/components/modal/index.tsx

```diff
@@ -24,7 +26,7 @@ import styles from './styles';
  * Uses styles defined on the <picker-field> element for the modal and buttons.
  * This is used on iOS only.
  */
-export default (props: Props): Node => {
+export default (props: Props): JSX.Element => {
```

Comment:
Do we want to use ReactNode here instead?

Reply:
https://github.com/Instawork/hyperview/pull/711#discussion_r1347725966